packages/devops/configs/servers.conf

```diff
@@ -3,7 +3,7 @@ server {
   listen       80;
   listen       [::]:80;
   server_name  ~^.*admin\.tupaia\.org$;
-  root         /home/ubuntu/tupaia/packages/admin-panel/served_build/;
+  root         /home/ubuntu/tupaia/packages/admin-panel/build;
```

Comment:
Now that we don't need to swap out the build on the server, we can use the more standard `build` directory so that there are less surprises